packages/x-scheduler-headless/src/use-event-occurrences-grouped-by-resource/useEventOccurrencesGroupedByResource.test.ts

```diff
@@ -40,34 +40,34 @@ describe('innerGetEventOccurrencesGroupedByResource', () => {
   it('should group single occurrence under a resource', () => {
     const R1 = makeResource('R1', 'Resource 1');
 
-    const occurrence = EventBuilder.new()
+    const event = EventBuilder.new()
```

Comment:
The tests were failing because we were passing an occurrence instead of a processed event as expected.